Update python.rst (#18)
AaltoSciComp/python-for-scicomp: python.rst

.. questions::

>    - Q1 What are the basic blocks of Python language?
>    - Q2 How are functions and classes defined in Python? 

Reviewer: ```suggestion
   - What are the basic blocks of Python language?
   - How are functions and classes defined in Python? 
```

.. objectives::

>    - O1 Get a *very* short introduction to Python types and syntax
>    - O2 ???

Reviewer: ```suggestion
   - Get a *very* short introduction to Python types and syntax
   - Be able to follow the rest of the examples in the course, even if you don't understand everything perfectly.
```

(typically) reified during compilation.

??? (lesson here)

.. keypoints::

>    - K1 Python offers a nice set of basic types as many other programming languages
>    - K2 Python is strongly typed and dynamically typed

Reviewer: ```suggestion
   - Python offers a nice set of basic types as many other programming languages
   - Python is strongly typed and dynamically typed
```
